cmake/PkgConfigWithFallback.cmake

```diff
@@ -13,11 +13,16 @@ function(find_pkg_config_with_fallback name)
         # Found via pkg-config, using it's result values
         set(${name}_FOUND ${${name}_PKG_CONFIG_FOUND})
 
+        if(MINGW)
+            list(FILTER ${name}_PKG_CONFIG_LIBRARIES EXCLUDE REGEX "(ole32)|(gdi32)|(imm32)|(shell32)|(winmm)|(dwmapi)|(setupapi)|(cfgmgr32)|(ws2_32)")
```

Comment:
This seems to be a hack specific to your build system. These libraries look like they actually are linker dependencies and thus should be included.

Reply:
Yes, it is a quirk of MSYS2. It is not needed when cross-compiling from Fedora.
Why does it need to manually find each DLL, by the way? Doesn't the linker know where they are anyway? 🤔

---

I changed the blacklist to explicity add the implicit library directory to the hint path, is this OK? Is there a better way?